### DISCUSSION TITLE
Sign-in to monarc trough AD

### DISCUSSION THREAD

**Discussion by Pyr0technicien** [monarc-project/MonarcAppFO#408] [Q&A] [answered]
Hi everyone,

Simple question, can't find any documentation about this.
Would there be a way to login to monarc through or AD ?

Thanks

**Comment by ruslanbaidan** [comment, marked as answer]
Hi,
For the moment it is not possible.
We are planning to implement it in the nearest future https://github.com/monarc-project/MonarcAppFO/issues/22
